packages/mobile/src/account/saga.ts

```diff
@@ -14,25 +13,13 @@ import { CustomEventNames } from 'src/analytics/constants'
 import { ErrorMessages } from 'src/app/ErrorMessages'
 import { navigate, navigateBack } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
-import { getPinFromKeystore, setPinInKeystore } from 'src/pincode/PhoneAuthUtils'
-import { getCachedPincode, setCachedPincode } from 'src/pincode/PincodeCache'
+import { getPinCache } from 'src/pincode/PasswordCache'
 import Logger from 'src/utils/Logger'
 
 const TAG = 'account/saga'
 
-export function* setPincode({ pincodeType, pin }: SetPincodeAction) {
+export function* setPincode({ pincodeType }: SetPincodeAction) {
   try {
-    if (pincodeType === PincodeType.PhoneAuth) {
-      Logger.debug(TAG + '@setPincode', 'Setting pincode with using system auth')
-      pin = randomBytes(10).toString('hex') as string
-      yield call(setPinInKeystore, pin)
-    } else if (pincodeType === PincodeType.CustomPin && pin) {
```

Comment:
with this gone, what's this function supposed to be doing now?

Reply:
I left it so that we could easily add biometrics in later